src/book.py

```diff
@@ -180,39 +217,79 @@ def tokenize(self, on, rem_stopwords=True, stopword_lang='english',
                     str.maketrans('', '', string.punctuation + '”“’'))
         return token
 
-    def snippet(self, length, on, non_cont=False, inplace=False):
+    def snippet(self, length, on, groups=1, non_cont=False, randomized=False, rem_stopwords=False, ret_as_arr=False, with_replace=True, random_seed=0, inplace=False):
```

Comment:
random_seed's default should be set to `None` rather than to 0 as 0 is a valid seed. Using `None` also aligns with what the random.seed() method uses as its default.

Reply:
Ok.